spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-reactive-sample/README.adoc

```diff
@@ -13,16 +13,17 @@ Alternatively, if you have the https://cloud.google.com/sdk/[Google Cloud SDK] i
 3. Still in the same page, locate the newly created topic, click the button with the three vertical dots at the end of the topic's line and click "New subscription".
 Create a new subscription called `exampleSubscription` with all default parameters.
 
-4. In a terminal window, move into this directory (spring-cloud-gcp-pubsub-reactive-sample) and run: `mvn spring-boot:run`.
+4. In a terminal window, move into this directory (`spring-cloud-gcp-pubsub-reactive-sample`) and run:
 
-5. In another terminal, send a message to the GCP Pub/Sub topic.
-Observe that nothing is received by the sample application, since there are no subscribers yet.
+  mvn spring-boot:run
 
-  gcloud pubsub topics publish exampleTopic --message "hello!"
+5. In a browser, open http://localhost:8080, and send some (`N`) messages to the GCP Pub/Sub topic.
+The messages sent will all start with the message text provided, and end with a number from `0` to `N-1`.
+Observe that nothing is received by the sample application, since there are no subscribers yet.
 
-6. In a third terminal, open a connection to '/getmessages' streaming endpoint.
+6. In another terminal, open a connection to `/getmessages` streaming endpoint.
 
   curl localhost:8080/getmessages
 
 
-7. Verify that the `curl` command got the message text, while the application itself logged its message ID, matching the one returned by `gcloud` command.
\ No newline at end of file
+7. Verify that the `curl` terminal received the messages, while the application itself logged their IDs.
```

Comment:
You might want to actually make this step before (6), and tell the user to watch for messages to appear after publishing.

Reply:
I'd rather add another step to publish more messages. I think it's an important point that no messages are retrieved until there is a subscription.